tsconfig.json

```diff
@@ -1,34 +1,23 @@
-// This file is only here for usage by our IDE. The tsconfig that angular really uses is in `tsconfig.base.json`. ´
-// More information: https://angular.io/guide/migration-solution-style-tsconfig
+/*
+  This is a "Solution Style" tsconfig.json file, and is used by editors and TypeScript’s language server to improve development experience.
+  It is not intended to be used to perform a compilation.
+
+  To learn more about this file see: https://angular.io/config/solution-tsconfig.
+*/
 {
-  "compileOnSave": false,
-  "compilerOptions": {
-    "baseUrl": "./",
-    "outDir": "./dist/out-tsc",
-    "sourceMap": true,
-    "declaration": false,
-    "downlevelIteration": true,
-    "experimentalDecorators": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "importHelpers": true,
-    "target": "es2015",
-    "typeRoots": [
-      "node_modules/@types"
-    ],
-    "lib": [
-      "es2018",
-      "dom"
-    ],
-    "paths": {
-      "@dlr-eoc/*": [
-        "dist/*",
-        "projects/*"
-      ]
+  "files": [],
```

Comment:
The content I added in the last comment was only a example...
So this should look now like:
```json
/*
  This is a "Solution Style" tsconfig.json file, and is used by editors and TypeScript’s language server to improve development experience.
  It is not intended to be used to perform a compilation.
  To learn more about this file see: https://angular.io/config/solution-tsconfig.
*/
{
  "files": [],
  "references": [
    {
      "path": "./tsconfig.app.json"
    },
    {
      "path": "./tsconfig.spec.json"
    },
    {
      "path": "./projects/base-layers-raster/tsconfig.lib.json"
    },
    {
      "path": "./projects/base-layers-raster/tsconfig.spec.json"
    },
    {
      "path": "./projects/demo-maps/tsconfig.app.json"
    },
    {
      "path": "./projects/demo-maps/tsconfig.spec.json"
    },

   // and the rest of our projects
   ... 
  ]
}
```

If you run this command `ng update @angular/cli --from 9 --to 10.0.4 --migrateOnly=true` it will make all the migrations, but unfortunately replaces our "tslib": "0.0.0-PLACEHOLDER-VENDOR" in the libraries package.json so this have to be reverted.

Reply:
You can also add this to the tsconfig.base.json, it should add [intellisense support and type checking for openlayers](https://www.npmjs.com/package/ol/v/6.4.0#user-content-intellisense-support-and-type-checking-for-vs-cod)

```json
{
...
"compilerOptions": {
    ...
    "paths": {
      ...
      "ol": [
        "node_modules/ol/src"
      ],
      "ol/*": [
        "node_modules/ol/src/*"
      ]
    },
    "allowJs": true,
  },
  "include": [
    "**/*.ts",
    "node_modules/ol/**/*"
  ],
  "typeAcquisition": {
    "exclude": [
      "ol"
    ]
  },
...
}
```